backend/spec/factories/users.rb

```diff
@@ -4,15 +4,20 @@
 #
 # Table name: users
 #
-#  id                             :bigint           not null, primary key
-#  email(ユーザーのEmailアドレス) :string(255)      not null
-#  password(ユーザーのpassword)   :string(255)      not null
-#  created_at                     :datetime         not null
-#  updated_at                     :datetime         not null
+#  id                                  :bigint           not null, primary key
+#  account_name(ユーザーの名前)        :string(60)       not null
+#  admin(管理者フラグ)                 :boolean          default(FALSE), not null
+#  password_digest(ユーザーのpassword) :string(60)       not null
+#  created_at                          :datetime         not null
+#  updated_at                          :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_account_name  (account_name) UNIQUE
 #
 FactoryBot.define do
   factory :user do
-    email { 'hogehoge@example.com' }
-    password { 'sample_password' }
+    account_name { 'test_user' }
```

Comment:
Nits

unique制約があると思うので、以下のようにsequenceを使って被らないようにしましょう！

```suggestion
    sequence(:account_name) { "User-#{_1}" }
```